app/controllers/expense_categories_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ExpenseCategoriesController < ApplicationController
+  before_action :set_expense_category, only: %i[show edit update destroy]
+
+  before_action :authenticate_user!
+
+  def index
+    @category = Category.find(params[:category_id])
+    @expenses = @category.expense_categories.includes(:expense).map(&:expense)
```

Comment:
- [ ] On the list of expenses in a category the expenses should be ordered by most recent. Kindly do that. You can append `.order(created_at: :desc)` after `includes(:expense)`

  ```
  @expenses = @category.expense_categories.includes(:expense).order(created_at: :desc).map(&:expense)
  ```